Formula/consul.rb

```diff
@@ -35,7 +35,18 @@ def install
     end
   end
 
-  plist_options :manual => "consul agent -dev -advertise 127.0.0.1"
+  def caveats
+    <<~EOS
+      The plist included with this formula runs Consul in -dev mode. This is "an
+      in memory server mode with some common features enabled (despite security
+      risks) for ease of use, and all persistence options turned off".
+
+      To learn how to run in server mode, with persistence, see the docs at:
+        https://learn.hashicorp.com/consul/getting-started/join#start-the-agents
+    EOS
+  end
```

Comment:
Since this is the option recommended by upstream I don't think we should have an explanation here. This should be enough
```suggestion
  # -dev is the suggested default by upstream
```